### PR TITLE
[KARAF-6593] Replace references in client script to slf4j-api bundle jar with pax-logging-api and remove it from assembly. Set version of slf4j-api to 1.7.29 in parent pom.

### PR DESCRIPTION
**Pull request by graben**
@jbonofre : Plz have a look on this.

### CODE REVIEW

File: assemblies/features/base/src/main/filtered-resources/resources/bin/client [hunk 1]
```diff
@@ -125,16 +125,16 @@ init() {
...
-        -Dkaraf.instances="${KARAF_HOME}/instances" \
-        -Dkaraf.home="${KARAF_HOME}" \
-        -Dkaraf.base="${KARAF_BASE}" \
-        -Dkaraf.etc="${KARAF_ETC}" \
-        -Dkaraf.log="${KARAF_LOG}" \
-        -Djava.io.tmpdir="${KARAF_DATA}/tmp" \
-        -Djava.util.logging.config.file="${KARAF_BASE}/etc/java.util.logging.properties" \
-        ${KARAF_OPTS} ${OPTS} \
-        -classpath "${CLASSPATH}" \
+    exec "${JAVA}" ${JAVA_OPTS} /
```
Comment (jbonofre) [inline assemblies/features/base/src/main/filtered-resources/resources/bin/client]: I guess it's unexpected right ?

Comment (graben) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/client]: Yes, too many replaces of backslashes. :-)

**Comment by jbonofre** [comment]
retest this please

**Comment by jbonofre** [comment]
I'm doing a final test pass and I will merge. Thanks !

**Comment by graben** [comment]
The client bundle has "org.slf4j.impl" as private-package and therefore embeds a few slf4j classes.

**Comment by jbonofre** [comment]
so, I will embed all required for logging then.

**Comment by graben** [comment]
@jbonofre : I'll look into code dealer this afternoon. Maybe I found something better.

**Comment by jbonofre** [comment]
@graben awesome ! Thanks ! Let me know if I can help.

File: pom.xml [hunk 2]
```diff
@@ -292,7 +292,7 @@
 
         <portlet-api.version>2.0</portlet-api.version>
         <plexus-utils.version>3.0.24</plexus-utils.version>
-        <slf4j.version>1.7.12</slf4j.version>
+        <slf4j.version>1.7.29</slf4j.version>
```
Comment (MrEasy) [inline pom.xml]: There is 1.7.30 in the meantime

**Comment by MrEasy** [comment]
slf4j:1.7.30:
http://mailman.qos.ch/pipermail/announce/2019/000153.html

**Comment by graben** [comment]
@MrEasy : pax-logging is still linked against 1.7.29 :-)

**Comment by MrEasy** [comment]
> @MrEasy : pax-logging is still linked against 1.7.29 :-)

Ah, didn't say anything then ;-)

**Comment by graben** [comment]
retest this please

**Comment by graben** [comment]
@jbonofre : Embedding api is the best solution I found. Not very nice but I think a little bit better. ;-)

**Comment by jbonofre** [comment]
retest this please

**Pull request merged**